actors/evm/src/lib.rs

```diff
@@ -189,14 +192,14 @@ impl EvmContractActor {
     pub fn handle_filecoin_method<RT>(
         rt: &mut RT,
         method: u64,
-        codec: u64,
-        params: &[u8],
+        args: Option<IpldBlock>,
     ) -> Result<Vec<u8>, ActorError>
     where
         RT: Runtime,
         RT::Blockstore: Clone,
     {
-        let input = handle_filecoin_method_input(method, codec, params);
+        let params = args.unwrap_or(IpldBlock { codec: 0, data: vec![] });
```

Comment:
Note: _this_ use of 0 for the codec is fine, because it's the only sane way to pass this information through to the contract.

Reply:
Yup, it all makes sense now.